core/put/source/dev_ui.h

```diff
@@ -63,9 +63,11 @@ namespace put
                 texture_cube_array = pen::TEXTURE_COLLECTION_CUBE_ARRAY
             };
         }
-        typedef e_ui_shader::ui_shader_t ui_shader;
 
-        bool        init();
+        typedef e_ui_shader::ui_shader_t ui_shader;
```

Comment:
`typedef e_ui_shader::ui_shader_t ui_shader;` should be beneath the enum with no space and there should be a space before the init function.

Please refer to the previous code for the formatting. the detail is subtle but important